registry.json

```diff
@@ -1,4 +1,19 @@
 [
+   {
+      "ticker":"NEXUS",
+      "poolId":"2b5ac5073e3e8071024263a8fe69e4577bcf3bfed41110ced8ac8aa7",
+      "social":{
+         "facebook":"",
+         "twitter":"NexusStake",
+         "website":"https://nexuspool.org",
+         "reddit":"",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"NEXUSCardanoStakePool",
```

Comment:
Please provide a Telegram username.  We use Telegram for voting purposes and we use your username to let you in the SPA voting group.

Reply:
Done! Thanks!